packages/playwright-test/src/poolBuilder.ts

```diff
@@ -15,25 +15,29 @@
  */
 
 import { FixturePool, isFixtureOption } from './fixtures';
+import type { LoadError } from './fixtures';
 import type { Suite, TestCase } from './test';
 import type { TestTypeImpl } from './testType';
 import type { Fixtures, FixturesWithLocation, FullProjectInternal } from './types';
+import { formatLocation } from './util';
 
 export class PoolBuilder {
   private _project: FullProjectInternal | undefined;
   private _testTypePools = new Map<TestTypeImpl, FixturePool>();
   private _type: 'loader' | 'worker';
+  private _loadErrors: LoadError[] | undefined;
 
-  static buildForLoader(suite: Suite) {
-    new PoolBuilder('loader').buildPools(suite);
+  static buildForLoader(suite: Suite, loadErrors: LoadError[]) {
+    new PoolBuilder('loader', loadErrors).buildPools(suite);
   }
 
   static createForWorker(project: FullProjectInternal) {
-    return new PoolBuilder('worker', project);
+    return new PoolBuilder('worker', undefined, project);
```

Comment:
who is supposed to handle the errors in the worker? are they propagated to the runner with a global catch or something?

Reply:
Errors in the worker throw in place, they are not collected.